internal/venice-test-common/src/main/java/com/linkedin/venice/utils/TestChunkingUtils.java

```diff
@@ -0,0 +1,183 @@
+package com.linkedin.venice.utils;
+
+import static com.linkedin.venice.kafka.protocol.enums.MessageType.*;
+
+import com.linkedin.venice.kafka.protocol.Delete;
+import com.linkedin.venice.kafka.protocol.GUID;
+import com.linkedin.venice.kafka.protocol.KafkaMessageEnvelope;
+import com.linkedin.venice.kafka.protocol.ProducerMetadata;
+import com.linkedin.venice.kafka.protocol.Put;
+import com.linkedin.venice.kafka.protocol.Update;
+import com.linkedin.venice.kafka.protocol.enums.MessageType;
+import com.linkedin.venice.message.KafkaKey;
+import com.linkedin.venice.pubsub.ImmutablePubSubMessage;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import com.linkedin.venice.pubsub.api.PubSubTopicPartition;
+import com.linkedin.venice.serialization.KeyWithChunkingSuffixSerializer;
+import com.linkedin.venice.serialization.avro.AvroProtocolDefinition;
+import com.linkedin.venice.serialization.avro.ChunkedValueManifestSerializer;
+import com.linkedin.venice.storage.protocol.ChunkId;
+import com.linkedin.venice.storage.protocol.ChunkedKeySuffix;
+import com.linkedin.venice.storage.protocol.ChunkedValueManifest;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+
+
+public class TestChunkingUtils {
+  private TestChunkingUtils() {
+    // Util class
+  }
+
+  public final static int CHUNK_LENGTH = 10;
+  public final static KeyWithChunkingSuffixSerializer keyWithChunkingSuffixSerializer =
```

Comment:
nit: static identifiers typically have capitalized name. not sure if this is brought up as warning by spotBugs.
e.g., 
s/keyWithChunkingSufficSerializer/`KEY_WITH_CHUNKING_SUFFIX_SERIALIZER`

Reply:
Fixed in 53008a1. I just tried `spotBugs`, and I didn't see any warning / failure about the naming.